.github/workflows/release.yml

```diff
@@ -23,7 +23,6 @@ jobs:
           node-version: 14
           cache: 'yarn'
       - run: git config user.email "-" && git config user.name "🤖 Release Bot"
-      - run: echo //registry.npmjs.org/:_authToken=${NPM_TOKEN} > .npmrc
-      - run: yarn install
-      - run: yarn lerna version --yes --conventional-commits --create-release github ${{ github.event.inputs.releaseParams }}
+      - run: yarn install --frozen-lockfile
+      - run: yarn lerna version --yes --conventional-commits --create-release github ${{ github.event.inputs.releaseParams }} --registry=https://registry.npmjs.org/:_authToken=${NPM_TOKEN}
```

Comment:
The `--registry` flag exists. So let's try it 🤷‍♂️ 

https://github.com/lerna/lerna/blob/6cb8ab2d4af7ce25c812e8fb05cd04650105705f/commands/create/README.md